code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -69,33 +67,188 @@
 	if (!check_and_use_plasma_owner())
 		return
 
-	if (xeno.mutation_type != LURKER_NORMAL)
+	if (Xeno.mutation_type != LURKER_NORMAL)
 		return
 
-	var/datum/behavior_delegate/lurker_base/behavior = xeno.behavior_delegate
+	var/datum/behavior_delegate/lurker_base/behavior = Xeno.behavior_delegate
 	if (istype(behavior))
 		behavior.next_slash_buffed = TRUE
 
-	to_chat(xeno, SPAN_XENOHIGHDANGER("Your next slash will deal increased damage!"))
-	button.icon_state = "template_active"
+	to_chat(Xeno, SPAN_XENOHIGHDANGER("Your next slash will deal increased damage!"))
 
 	addtimer(CALLBACK(src, .proc/unbuff_slash), buff_duration)
-	xeno.next_move = world.time + 1 // Autoattack reset
+	Xeno.next_move = world.time + 1 // Autoattack reset
 
 	apply_cooldown()
 	..()
 	return
 
 /datum/action/xeno_action/onclick/lurker_assassinate/proc/unbuff_slash()
-	var/mob/living/carbon/Xenomorph/xeno = owner
-	if (!istype(xeno))
+	var/mob/living/carbon/Xenomorph/Xeno = owner
+	if (!istype(Xeno))
 		return
-	var/datum/behavior_delegate/lurker_base/behavior = xeno.behavior_delegate
+	var/datum/behavior_delegate/lurker_base/behavior = Xeno.behavior_delegate
 	if (istype(behavior))
 		// In case slash has already landed
 		if (!behavior.next_slash_buffed)
 			return
 		behavior.next_slash_buffed = FALSE
 
-	to_chat(xeno, SPAN_XENODANGER("You have waited too long, your slash will no longer deal increased damage!"))
-	button.icon_state = "template"
+	to_chat(Xeno, SPAN_XENODANGER("You have waited too long, your slash will no longer deal increased damage!"))
+
+
+// VAMPIRE LURKER
+
+/datum/action/xeno_action/activable/pounce/rush/additional_effects(mob/living/L) //pounce effects
```

Comment:
i would rather this was a descriptive var name but it's only used right next to declaration so... erm...